src/main/java/com/gigacal/controller/CalendarController.java

```diff
@@ -24,9 +20,38 @@ public ResponseEntity<?> getCalendar() {
         return ResponseEntity.status(HttpStatus.OK).body(calendarService.findAll());
     }
 
+    @GetMapping(path = "{id}")
+    public ResponseEntity<?> getCalendarById(@PathVariable("id") Long id) {
+        return ResponseEntity.status(HttpStatus.OK).body(calendarService.findCalendarById(id));
```

Comment:
Missed this. -> applies to all of the places as there are many of them and I don't want to spam this comment :)